dev-util/cutter/patches/cutter-1.10.2.patchset

```diff
@@ -0,0 +1,31 @@
+From 90350b09d5e5f3b8149f46f93af1f39129ee77fa Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Zolt=C3=A1n=20Mizsei?= <zmizsei@extrowerk.com>
+Date: Fri, 3 Apr 2020 10:50:38 +0200
+Subject: Build fix
+
+
+diff --git a/src/widgets/ConsoleWidget.cpp b/src/widgets/ConsoleWidget.cpp
+index b1e1024..267457e 100644
+--- a/src/widgets/ConsoleWidget.cpp
++++ b/src/widgets/ConsoleWidget.cpp
+@@ -441,7 +441,7 @@ void ConsoleWidget::redirectOutput()
+     pipe(redirectPipeFds);
+     stdinFifoPath = QString(STDIN_PIPE_NAME).arg(QDir::tempPath(), QUuid::createUuid().toString());
+     mkfifo(stdinFifoPath.toStdString().c_str(), (mode_t) 0777);
+-    stdinFile = open(stdinFifoPath.toStdString().c_str(), O_RDWR | O_ASYNC);
```

Comment:
I'm not quite sure about this. `O_ASYNC` is an actual flag that send signals to the program when the file descriptor is ready to be read from. Removing it might break the code in subtle ways.

Reply:
But it is undeclared

Reply:
Because Haiku doesn't support it, but the application might depend on it.

Reply:
Looks like we are not alone in this? (checking up on cutter 1.12.0 reveals the same thing)
```diff
         int fd = qemu_open(path, O_RDWR | O_NONBLOCK
-#ifndef CONFIG_SOLARIS
+#if !defined(CONFIG_SOLARIS) && !defined(__MSYS__)
                            | O_ASYNC
 #endif
                            );
```